packages/core-modules/contracts/interfaces/IElectionModule.sol

```diff
@@ -28,7 +28,7 @@ interface IElectionModule {
 
     function setNextPeriodPercent(uint8 percent) external;
 
-    function elect(address[] memory candidates) external;
+    function elect(address[] memory sortedCandidates, uint[] memory positions) external;
```

Comment:
Add documentation to function with additional info about the parameters? For example if positions is 0 indexed or 1 indexed.

Reply:
Done: https://github.com/Synthetixio/synthetix-v3/pull/572/files#diff-69650d386e562b4dac992709531657fe63fedd176cde00f236b7b6271333c70eR144-R147